apis/duck/v1alpha1/legacy_targetable_types.go

```diff
@@ -24,19 +24,19 @@ import (
 	"github.com/knative/pkg/apis/duck"
 )
 
-// LegacyTargetable left around until we migrate to Targetable in the
-// dependent resources. Targetable has more structure in the way it
+// LegacyTargetable left around until we migrate to Addressable in the
+// dependent resources. Addressable has more structure in the way it
 // defines the fields. LegacyTargetable only assumed a single string
 // in the Status field and we're moving towards defining proper structs
 // under Status rather than strings.
 // This is to support existing resources until they migrate.
 //
-// Do not use this for anything new, use Targetable
+// Do not use this for anything new, use Addressable
 
-// LegacyTargetable is the old schema for the targetable portion
+// LegacyTargetable is the old schema for the addressable portion
 // of the payload
 //
-// For new resources use Targetable.
+// For new resources use Addressable.
```

Comment:
If we have two things that are legacy, let's just get rid of this at least?

Reply:
I think we want to keep the legacy_ version, since 0.1 only implements the Legacy version. I'd rather ditch the abortive naming that's only in 0.2.0.

Reply:
`Flow` still uses domainInternal, not that I expect it to live long, but for the moment it's a thing.